Samples/iOS-Swift/iOS-SwiftUITests/BaseUITest.swift

```diff
@@ -1,26 +1,44 @@
 import XCTest
 
 class BaseUITest: XCTestCase {
-    internal let app: XCUIApplication = XCUIApplication()
+    internal lazy var app: XCUIApplication = newAppSession()
+    
+    //swiftlint:disable implicit_getter
+    var automaticallyManageAppSession: Bool { get { true } }
```

Comment:
`l`: Maybe a bit clearer
```suggestion
    var automaticallyLaunchAndTerminateApp: Bool { get { true } }
```